### PR TITLE
align with gen-signedexchange

### LINKED ISSUE #23: Align flags with webpackage signedexchange
For example, gensxg uses `-url` instead of `-uri` and does not support `-o`.

### PR DESCRIPTION
**Pull request by kumagi**
closes #23 
flags are aligned.

### CODE REVIEW

File: src/gensxg.c [hunk 1]
```diff
@@ -449,23 +458,34 @@ static void print_integrity_hash(const sxg_encoded_response_t* encoded) {
...
+
+  FILE* out;
+  if (strcmp(output, "-") == 0) {
+    out = freopen(NULL, "wb", stdout);
+    if (out == NULL) {
+      perror("reopen stdout");
+      exit(EXIT_FAILURE);
+    }
+  } else {
+    out = fopen(output, "wb");
```
Comment (yuizumi) [inline src/gensxg.c]: I believe you need to close this file.

File: src/gensxg.c [hunk 2]
```diff
@@ -449,23 +458,34 @@ static void print_integrity_hash(const sxg_encoded_response_t* encoded) {
...
     exit(EXIT_FAILURE);
   }
-    
-  if (freopen(NULL, "wb", stdout) == NULL) {
-    perror("reopen stdout");
-    exit(EXIT_FAILURE);
+
+  FILE* out;
+  if (strcmp(output, "-") == 0) {
+    out = freopen(NULL, "wb", stdout);
```
Comment (gaul) [inline src/gensxg.c]: Is this actually required or would `out = stdout` suffice?  If it is required, presumably to open as binary, please add a comment.

Comment (kumagi) [reply src/gensxg.c]: In linux environment, `stdout` is binary mode by default, so it is not actually required.
But other environment may be different, so priority was given to clarity.

File: src/gensxg.c (hunk 1 above)
Comment (kumagi) [reply src/gensxg.c]: OK, I'll close.

Comment (kumagi) [reply src/gensxg.c]: https://github.com/google/libsxg/pull/33
created PR.

**Pull request merged**